ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.WebAPI/Controllers/ProjectsController.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace ExadelTimeTrackingSystem.WebAPI.Controllers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using ExadelTimeTrackingSystem.BusinessLogic.DTOs;
+    using ExadelTimeTrackingSystem.BusinessLogic.Services.Abstract;
+    using Microsoft.AspNetCore.Mvc;
+
+    [ApiController]
+    [Route("[controller]")]
+    public class ProjectsController : ControllerBase
+    {
+        private readonly IProjectService _service;
+
+        public ProjectsController(IProjectService service)
+        {
+          _service = service;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<List<ProjectDTO>>> GetAllAsync()
+        {
+           var projects = await _service.GetAllAsync();
+           return Ok(projects);
+        }
+
+        [HttpGet]
```

Comment:
I can predict that your app throws error when you're launching the app, so review articles about WebAPI and see what you missed